user_front/pages/regist/group/index.vue

```diff
@@ -1,10 +1,39 @@
-<script lang="ts" setup>
+<script lang="ts">
 </script>
 
 <template>
   <div>
     <div class="mx-[20%] my-[5%]">
-      <Card>Registration of organization</Card>
+      <Card>
+        <h1 class="text-3xl">Registration of organization</h1>
+        <Card border="none" align="end" gap="20px">
+          <div class="flex">
+            <p class="flex-none text-xl pr-5">name</p>
+            <input class="flex-none border-solid border-2" />
+          </div>
+          <div class="flex">
+            <p class="flex-none text-xl  pr-5">student id</p>
+            <input class="flex-none border-solid border-2" />
+          </div>
+          <div class="flex">
+            <p class="flex-none text-xl  pr-5">department</p>
+            <input class="flex-none border-solid border-2" />
+          </div>
+          <div class="flex">
+            <p class="flex-none text-xl  pr-5">grade</p>
+            <input class="flex-none border-solid border-2" />
+          </div>
+          <div class="flex">
+            <p class="flex-none text-xl pr-5">mail adress</p>
+            <input class="flex-none border-solid border-2" />
+          </div>
+          <div class="flex">
+            <p class="flex-none text-xl pr-5">phone number</p>
+            <input class="flex-none border-solid border-2" />
+          </div>
```

Comment:
```suggestion
<script lang="ts" setup>
  const formTitle = [
    "name",
    "student id",
    "department",
    "grade",
    "mail adress",
    "phone number",
  ];
</script>
         ...
<div v-for="title in formTitle" :key="title" class="flex">
   <p class="label">{{ title }}</p>
   <input class="form" />
</div>
```
同じ処理してるのでこんな感じにかけます。